docker/config.yaml.mako

```diff
@@ -121,6 +121,9 @@ vars:
       # Specify any additional URL parameters that the print shall use for WMS calls
       wms_url_params:
         TRANSPARENT: 'true'
+      # Set this to true fi all parameters specified in the original URL (from the DB) should be reused in the map
+      # request for the restriction_on_landownership baseLayers.
+      wms_url_params_from_DB: false
```

Comment:
We should remove the "from the DB" part. It does not necessarily come from the DB (it comes from the DB if the default implementation is used, but it could come from anywhere)

How about: 
wms_url_keep_params